### ISSUE TITLE
TimeFrame property name

### ISSUE THREAD

**Issue by afolarin** [RADAR-CNS/RADAR-RestApi#38]
The enum param “timeFrame” name is confusing and should be changed to “timeInterval”. 

OMH does have a “time_interval” property to define the time interval, although their definition is different from ours:
 - http://www.openmhealth.org/documentation/#/schema-docs/schema-library/schemas/omh_time-frame 
 - http://www.openmhealth.org/documentation/#/schema-docs/schema-library/schemas/omh_time-interval

```
"timeFrame": "TEN_SECOND",
should be
"timeInterval": "TEN_SECOND",
```

TODO
- [x] rename response `timeFrame` to ~~`timeInterval`~~ 'timeWindow'
- [x] Fix inconsistency between the property name in the GET Request and the Response

**Comment by blootsvoets** [comment]
Note that in Kafka streams, where the data comes from, the `TEN_SECOND` symbol is called a _time window_. That is different from the time frame, since the `timeFrame` describes a dataset limited to a certain time frame. This latter name is consistent with the name _data frame_ used in analytics packages like pandas or R. Could that be a resolution? Call `TEN_SECOND` a `timeWindow` and keep `timeFrame` for the dataset time frame?

**Comment by herkulano** [comment]
`timeWindow` works well, the confusing name was `timeFrame`.

Regarding the `timeFrame` and `effectiveTimeframe`, I added a comment to clarify it here: https://github.com/RADAR-CNS/RADAR-RestApi/issues/37#issuecomment-331550793

**Comment by blootsvoets** [comment]
Fixed with RADAR-CNS/RADAR-Schemas#55.
